src/utils/encrypt-password.ts

```diff
@@ -0,0 +1,12 @@
+import * as bcrypt from 'bcrypt';
+
+export class EncryptPassword {
```

Comment:
```suggestion
export class PasswordEncripter {
```

Geralmente classes desse tipo a gente nomeia como substantivo. Verbos seriam mais pra funções.

Reply:
Perfeito, obrigado Alan!